cmd/libs/go2idl/client-gen/generators/client_generator.go

```diff
@@ -120,7 +120,7 @@ func packageForGroup(gv unversioned.GroupVersion, typeList []*types.Type, packag
 			return generators
 		},
 		FilterFunc: func(c *generator.Context, t *types.Type) bool {
-			return types.ExtractCommentTags("+", t.SecondClosestCommentLines)["genclient"] == "true"
+			return types.ExtractCommentTags("+", t.SecondClosestCommentLines)["genclient"] != nil
```

Comment:
Fixed in next push.  I added some tooling around this, though the evolution of deepcopy and conversion won't use it much, since they grew their own parsing.